notebooks/amazon_forest_notebook.py

```diff
@@ -190,119 +191,121 @@
 
 # <codecell>
 
-print("X_train/y_train lenght: {}/{}".format(len(preprocessor.X_train), len(preprocessor.y_train)))
-print("X_val/y_val lenght: {}/{}".format(len(preprocessor.X_val), len(preprocessor.y_val)))
-print("X_test/X_test_filename lenght: {}/{}".format(len(preprocessor.X_test), len(preprocessor.X_test_filename)))
+print("X_train/y_train length: {}/{}".format(len(preprocessor.X_train), len(preprocessor.y_train)))
+print("X_val/y_val length: {}/{}".format(len(preprocessor.X_val), len(preprocessor.y_val)))
+print("X_test/X_test_filename length: {}/{}".format(len(preprocessor.X_test), len(preprocessor.X_test_filename)))
 preprocessor.y_map
 
 # <markdowncell>
 
-# ## Callbacks
+# # Funetuning
 # 
-# 
-# 
-# __Checkpoint__
-# 
-# Saves the best model weights across all epochs in the training process.
-# 
-# __CSVLogger__
-# 
-# Creates a file with a log of loss and accuracy per epoch
-# 
-# __FBeta__
-# 
-# Calculates fbeta_score after each epoch during training
+# Here we define the model for finetuning
 
 # <codecell>
 
-checkpoint = ModelCheckpoint(filepath="weights.best.hdf5", monitor='val_acc', verbose=1, save_best_only=True)
+from keras.applications.vgg16 import VGG16
+from keras.models import Model
+from keras.layers import Dense, Input, Flatten, Dropout
+from keras.layers.normalization import BatchNormalization
 
-# creates a file with a log of loss and accuracy per epoch
-csv = CSVLogger(filename='training.log', append=True)
+# VGG16
 
-# Calculates fbeta_score after each epoch during training
-fbeta = Fbeta()
+img_dim = (128, 128, 3)
 
-# <markdowncell>
+input_tensor = Input(shape=img_dim)
 
-# ## Choose Hyperparameters
-# 
-# Choose your hyperparameters below for training. 
-# 
-# Note that we have created a learning rate annealing schedule with a series of learning rates as defined in the array `learn_rates` and corresponding number of epochs for each `epochs_arr`. Feel free to change these values if you like or just use the defaults.
-# 
-# If you opted for a high `img_resize` then you may want to lower the `batch_size` to fit your images matrices into the GPU memory. With an `img_resize` of `(256, 256)` (the default size) and a batch_size of `64` you should at least have a GPU with 8GB or VRAM.
+base_model = VGG16(include_top=False,
+                   weights='imagenet',
+                   input_shape=img_dim)
 
-# <codecell>
+bn = BatchNormalization()(input_tensor)
+x = base_model(bn)
+x = Flatten()(x)
+output = Dense(17, activation='sigmoid')(x)
+
+model = Model(input_tensor, output)
```

Comment:
We could move all this (line 218 to 227) to its own file as you did with `densenet169.py`. We can name it `vgg16.py` perhaps? :)